swift-spaces-real-time-app/README.md

```diff
@@ -0,0 +1,23 @@
+# Mux Spaces SDK Sample
+
+This is an iOS and iPadOS app written in Swift demonstrating using the Spaces SDK to add real-time video and audio to your app.
+
+Note: The Spaces SDK is Beta software and may contain issues or missing functionality. We recommend against submitting apps based on it to the App Store.
+
+## Installation
+
+### Adding the SDK using Swift Package Manager
+
+You can add the SDK using Swift Package Manager via Xcode by following the steps described (here)[https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app]. Use this [Swift Package URL](https://github.com/muxinc/mux-spaces-sdk-swift-distribution) when adding the SDK in Xcode.
+
+### Adding the SDK manually
+
+Follow the steps here to add the SDK manually.
```

Comment:
It's unclear which steps to follow here.

Reply:
I've removed this section.